index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
Okay, then how `targets` field in babel-brunch options should act when user defines another preset? It should be ignored, obviously. Then should we add an additional option only for `env`?

@shvaikalesh what do you think?